corehq/apps/commtrack/templates/commtrack/manage/rebuild_stock_state.html

```diff
@@ -0,0 +1,41 @@
+{% extends "settings/base_template.html" %}
+{% load i18n %}
+{% load hq_shared_tags %}
+
+{% block main_column %}
+    <table class="table">
+        <tr>
+            <th>Case</th>
+            <th>Section</th>
+            <th>Product</th>
+            <th>Date</th>
+            <th>Type</th>
+            <th>Quantity</th>
+            <th>Stock on Hand</th>
+        </tr>
+    {% for stock_transaction in stock_transactions %}
+        <tr>
+            <td>
+                {% ifchanged %}{{ stock_transaction.case_id }}{% endifchanged %}
+            </td>
+            <td>
+                {% ifchanged %}
+                    <!--{{ stock_transaction.case_id }}-->
```

Comment:
Haha, yeah. I knew I'd be iterating on this so I've been kinda quick and dirty. This particular thing is now gone in the course of changes I made in https://github.com/dimagi/commcare-hq/pull/7678.